site/install.md

```diff
@@ -210,7 +211,7 @@ sudo yum install -y --enablerepo=epel parquet-devel # For Apache Parquet C++
 sudo yum install -y --enablerepo=epel parquet-glib-devel # For Parquet GLib (C)
 ```
 
-### C++ and GLib (C) Packages for MSYS2 (Unofficial)
+### C++ and GLib (C) Packages for MSYS2
 
 The official MSYS2 packages include [Apache Arrow C++ and GLib (C)
```

Comment:
I'll note that this seems in conflict: heading said "unofficial" but the sentence says "official"

Reply:
I meant this is "official MSYS2 packages" and isn't "official Apache Arrow packages".
"official MSYS2 packages" means that they are included in https://github.com/msys2/MINGW-packages .

Anyway, we can remove the "official".